lib/services/causes_service.dart

```diff
@@ -93,11 +83,9 @@ class CausesService {
   /// Input Action id
   /// Returns the Action with that id
   Future<Campaign> getCampaign(int id) async {
-    final (response, userInfo) = await (
-      _causeServiceClient.getCampaignsApi().campaignsRetrieve(id: id),
-      getUserInfo(),
-    ).wait;
-    return Campaign.fromApiModel(response.data!, userInfo);
+    final response =
+        await _causeServiceClient.getCampaignsApi().campaignsRetrieve(id: id);
+    return Campaign.fromApiModel(response.data!);
```

Comment:
What if `response` will be `null`, shouldn't we return an error then?

Reply:
In theory an exception will get thrown which is caught a the bloc layer. I admit this isn't the best though. Do you think there is a clearer way of handling errors at the service level?